### PR TITLE
adds readme

### PR DESCRIPTION
**Pull request by Nirvikalpa108**
This PR adds details of the conference for potential speakers, mentors and attendees. This is just a first draft, so please feel free to suggest edits/make further PRs!💖

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,39 @@
-# scala-base-conf
-Scala Base Conference
+# Scalabase Conf 14 May 2021
```
Comment (frankie297) [inline README.md:1]: Have we decided this is the way we want to write it, not Scala Base?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,39 @@
...
+ScalaBase is a new mini-conference that will run on the 14th of May, right before this year's ScalaCon.
```
Comment (frankie297) [inline README.md]: You have used Scalabase most other places and here you used ScalaBase

**Review by frankie297 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Nirvikalpa108) [reply README.md:1]: I have no idea 😂

File: README.md (hunk 2 above)
Comment (Nirvikalpa108) [reply README.md]: Thank you so much for spotting this!❤️ I'll use Scalabase throughout. If we want to change it later, we always can 😊

**Pull request merged**